test/edu/stanford/dlss/was/TestWasapiDownloader.java

```diff
@@ -5,22 +5,20 @@
 import static org.junit.Assert.*;
 import org.junit.*;
 
-import edu.stanford.dlss.was.WasapiDownloader;
-
 public class TestWasapiDownloader {
 
     @Before
     public void setUp() {
     }
 
     @Test
-    public void constructorLoadsSettings() throws IOException {
+    public void constructor_loadsSettings() throws LoadSettingsException {
       WasapiDownloader myInstance = new WasapiDownloader(WasapiDownloader.SETTINGS_FILE_LOCATION, null);
       assertNotNull(myInstance.settings);
     }
 
     @Test
-    public void canExecuteMainWithoutCrashing() throws IOException {
+    public void main_canExecuteWithoutCrashing() throws LoadSettingsException {
       WasapiDownloader.main(null);
     }
```

Comment:
should there be tests for "run()" method, with both values for settings.shouldDisplayHelp() ?